### PR TITLE
Use a "TL;DR" style for displaying help

### PR DESCRIPTION
**Pull request by thekid**
See http://tldr-pages.github.io/
## Old

``` sh
$ unittest -?
Unittest command
----------------

Usage:

$ xp test [options] test [test [test...]]


Options is one of:

  -v : Be verbose
  -q : Be quiet (no output)
  -a {argument}: Define argument to pass to tests (may be used
    multiple times)
  -l {listener.class.Name} {output} [-o option value [-o ...]]
    where output is either - for console output or a file name.
    Options with -o are listener-dependant arguments.
  --color={mode} : Enable / disable color; mode can be one of
    . on - activate color mode
    . off - disable color mode
    . auto - try to determine whether colors are supported and enable
      accordingly.
  -w {dir}: Run indefinitely, watch given directory for changes
  -s {when}: Stop running when a certain event occurs. When may be:
    . fail - When the first test fails
    . skip - On the first skipped test
    . ignore - When the first ignored test is encountered

Tests can be one or more of:

  {tests}.ini: A configuration file
  {package.name}.*: All classes inside a given package
  {package.name}.**: All classes inside a given package and all subpackages
  {Test}.class.php: A class file
  {test.class.Name}: A fully qualified class name
  {test.class.Name}::{testName}: A fully qualified class name and a test name
  path/with/tests: All classes inside a given directory and all subdirectories
  -e {test method sourcecode}: Evaluate source
```
## New

tldr-test

_Note: The documentation for `--color` and `-l {listener.class.Name} {output} [-o option value [-o ...]]` has been omitted, as has been {tests}.ini, the non-recusrive variant {package.name}. as well as {test.class.Name}::{testName}. These are a bit special, and could be added to the Wiki here for example_

_The option `-w {/path/to/watch}` is superseded by the new XP runners global `-watch {/path/to/watch}` switch_

### CODE REVIEW

**Comment by thekid** [comment]
This is waiting for XP 6.10.0-RELEASE and maybe a bit for xp-runners/reference#10 which would have an answer to _...and could be added to the Wiki here for example._
